### PR TITLE
Implement shadow_offset

### PR DESCRIPTION
**Pull request by famfo**
I had to rebase the code so a new PR.

See: #225 
Fixes: #99

### CODE REVIEW

**Review by ErikReider — COMMENTED** [review]
LGTM! I'll wait for Will's approval before merging though

File: sway/sway.5.scd [hunk 1]
```diff
@@ -689,6 +689,9 @@ The default colors are:
 *shadow_inactive_color* <hex color with alpha>
 	The shadow color for inactive windows. Default value: *shadow_color*
 
+*shadow_offset* <x offset> <y offset>
+	Offset the default shadows by the given value.
```
Comment (WillPower3309) [inline sway/sway.5.scd:693]: Nit: wouldn't only apply to default shadows

File: sway/desktop/render.c [hunk 2]
```diff
@@ -342,15 +342,15 @@ void render_blur(bool optimized, struct sway_output *output,
...
+		float corner_radius, float offset_x, float offset_y) {
 	struct wlr_output *wlr_output = output->wlr_output;
 	struct fx_renderer *renderer = output->renderer;
 
 	struct wlr_box box;
 	memcpy(&box, _box, sizeof(struct wlr_box));
-	box.x -= blur_sigma;
-	box.y -= blur_sigma;
+	box.x -= blur_sigma - offset_x;
+	box.y -= blur_sigma - offset_y;
```
Comment (WillPower3309) [inline sway/desktop/render.c:353]: Q: wouldn't we want to add the offsets?

**Review by WillPower3309 — COMMENTED** [review]
One nit and one question! Otherwise LGTM!

Has this been tested on scaled and rotated outputs?

**Comment by ErikReider** [comment]
> Has this been tested on scaled and rotated outputs?

Tested and works :)

File: sway/desktop/render.c (hunk 2 above)
Comment (famfo) [reply sway/desktop/render.c:353]: When adding the offsets, the box gets moved into the opposite direction.

Comment (WillPower3309) [reply sway/desktop/render.c:353]: Ah yes, missed the double negative of subtracting from a subtracted value

**Comment by WillPower3309** [comment]
Awesome, thanks for the work here! Just want to give it a manual test and then I'll go ahead and merge it

**Review by WillPower3309 — APPROVED** [review]

**Pull request merged**